snuba/snuba_migrations/replays/0022_add_context_ota_updates.py

```diff
@@ -0,0 +1,67 @@
+from typing import Iterator, Sequence, Tuple
+
+from snuba.clickhouse.columns import Column, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+new_columns: Sequence[Tuple[Column[Modifiers], str]] = [
+    (
+        Column("ota_updates_channel", String(Modifiers(nullable=True))),
```

Comment:
If this has fewer than 1,000,000 unique values (globally) please wrap with `LowCardinality`.

Reply:
I'm going to merge without LowCardinality since the string is user specified.